backend/coins/btc/account.go

```diff
@@ -52,6 +51,8 @@ type Interface interface {
 	Info() *Info
 	Code() string
 	Coin() coin.Coin
+	// Name returns a human readable long name.
+	Name() string
```

Comment:
It would be nice to also add doc comment to the `Code` and explain how it's different from name. 

You already say that name is a "human readable long name" but what is code then, a "human readable short name" or a "machine-readable code or something else?

Reply:
At least, to me as an external viewer code and name seem to be related but I'm not sure what the difference is.

Reply:
Thx. Done!